features/steps/steps.py

```diff
@@ -0,0 +1,99 @@
+# pylint: disable=function-redefined, missing-function-docstring
+# flake8: noqa
+import logging
+import requests
+from datetime import date
+from behave import given, when, then
+from compare import expect, ensure
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select, WebDriverWait
+from selenium.webdriver.support import expected_conditions
+
+ID_PREFIX = "order_"
+
+@given('the following orders')
+def step_impl(context):
+    """ Delete all Orders and load new ones """
+    # List all of the orders and delete them one by one
+    rest_endpoint = f"{context.BASE_URL}/orders"
+    context.resp = requests.get(rest_endpoint)
+    expect(context.resp.status_code).to_equal(200)
+    for order in context.resp.json():
+        context.resp = requests.delete(f"{rest_endpoint}/{order['id']}")
+        expect(context.resp.status_code).to_equal(204)
+
+    # load the database with new pets
```

Comment:
change to comment to say orders instead of pets

Reply:
Will do